deploy/site.yaml

```diff
@@ -135,6 +135,11 @@ production:
     }
     more_set_headers "Link: <https://assets.ubuntu.com>; rel=preconnect; crossorigin, <https://assets.ubuntu.com>; rel=preconnect, <https://res.cloudinary.com>; rel=preconnect";
 
+  nginxServerSnippet: |
+    location /wp-content/ {
```

Comment:
I think we discussed that this should be explicitly limited to `/wp-content/uploads`, so that we're not exposing more to the outside world than we need. Do you agree? Could this be changed to that more explicit path?

Reply:
Yep. I will make the update it immediately.